.travis.yml

```diff
@@ -114,7 +107,7 @@ matrix:
             - qemu
             - gcc-arm-linux-gnueabihf
             - libc-dev-armhf-cross
-      env: CHOST=arm-linux-gnueabihf BUILDDIR=. TOOL="./configure --warn --zlib-compat --neon"
+      env: CHOST=arm-linux-gnueabihf BUILDDIR=. TOOL="./configure --warn --zlib-compat"
```

Comment:
> Travis config file needs to get it's --neon flag removed.

@Dead2 is this change enough to make the Travis CI tester pass, or is there another change that I need?

Reply:
I see that the tester just passed.  I don't know why the tester hasn't picked up this change yesterday.